binding_generator.py

```diff
@@ -230,6 +230,7 @@ def generate_builtin_class_header(builtin_api, size, used_classes, fully_used_cl
     result.append("namespace godot {")
     result.append("")
 
+    result.append(f"class Variant;")
```

Comment:
You don't need to forward-declare it here.
It will be handled automatically if we fix JSON generator. Because for now it generates `Nil` instead of `Variant` for newly added operators overloads for Variants.

Reply:
I see, thanks!